exercises/practice/sieve/.approaches/introduction.md

```diff
@@ -0,0 +1,37 @@
+# Introduction
+
+There are several variants of a general approach for solving Sieve.
+The general approach is to keep track of composite numbers with a slice of `bool` values.
+
+## General guidance
+
+Something to consider is to keep the number of allocations at a minimum to get the best performance.
+
+## Approach: Slice of `bool`s
+
+```go
+// Package sieve is a small library for finding prime numbers
+package sieve
+
+// Sieve calculates and returns prime numbers up to the limit
+func Sieve(limit int) []int {
+	composite := make([]bool, limit+1)
+	primes := make([]int, limit/2)
+	prime_index := 0
```

Comment:
Should be `primeIndex` (Official docs: https://go.dev/doc/effective_go#mixed-caps)

Reply:
Thanks!